frontend/src/react/Copy/AnswerExercice/AnswerQuestion/Question.js

```diff
@@ -15,10 +15,15 @@ class Question extends Component {
         editorState: EditorState.createEmpty(),
         id: '',
         answerId: '',
+        readOnly: false
     };
 
     componentDidMount() {
-        this.setState({id:this.props.id});
+        const answerFromProps = this.props.answer && !!this.props.answer.content;
```

Comment:
I have done the exact same thing in my PR, but another way. Shall we use the same method? Don't mind using yours, but since I needed to use the copy._id and the answer, I ended up having the whole copy object as a prop. 

Check checkForExistingAnswer().

Reply:
Sure ! My branch is not up to date with master so we might give a look at that when solving merge conflicts tomorrow ?

Reply:
Mmmmm, the only thing is that we do the exact same thing. Master is like this: https://github.com/EBM2018/iRate/blob/bee360765d46b7a2a4a9724c89fbc5efefee5f1a/frontend/src/react/Copy/AnswerExercice/AnswerQuestion/Question.js